openlibrary/macros/StarRatings.html

```diff
@@ -1,6 +1,9 @@
-$def with (work, edition=None, redir_url=None, id='')
+$def with (work, edition=None, redir_url=None, id='', rating=None)
+
 $ username = ctx.user and ctx.user.key.split('/')[-1]
-$ rating = work and username and work.get_users_rating(username)
+$# 0 can be passed in as a rating value for an unrated book, but it is falsey, so we can't use "if not rating" alone.
+$if not rating and rating != 0:
```

Comment:
Ahhhh, I see now! This should be the easiest way:

```suggestion
$if rating is None:
```